integr8ly/tutorial-web-app
# do not use simple git env

Fixes the `Out of Memory` error on startup that seems to be caused by a bug in simple git's `env` function.

Verification steps:

1. build an image with this branch (or use quay.io/integreatly/tutorial-web-app:pbraun) and deploy it to a cluster
2. The webapp should start normally and you should see a log entry saying that the walkthrough repo has been cloned
